packages/appkit/src/client.ts

```diff
@@ -684,6 +684,26 @@ export class AppKit {
     return null
   }
 
+  private setUnsupportedNetwork(chainId: string | number) {
+    const namespace = this.getActiveChainNamespace() || ConstantsUtil.CHAIN.EVM
```

Comment:
should we really default to evm? I mean, lets say someone is using only Solana adapter, would not this break?

Reply:
Agree, changed this